inc/Lazyload/Component.php

```diff
@@ -72,7 +72,8 @@ public function action_lazyload_images() {
 		}
 
 		add_action( 'wp_head', [ $this, 'action_add_lazyload_filters' ], PHP_INT_MAX );
-		add_action( 'wp_enqueue_scripts', [ $this, 'action_enqueue_lazyload_assets' ] );
+		add_action( 'wp_enqueue_scripts', [ $this, 'action_register_lazyload_assets' ] );
+		add_action( 'wp_head', [ $this, 'action_print_lazyload_script' ], 8 );
```

Comment:
This should be in `wp_print_footer_scripts` since it has to wait for DOM ready anyway. Better to not have the script block rendering in the head.

Reply:
Done via https://github.com/wprig/wprig/pull/577/commits/b888e3ce53cb41a4a170e48d4dd0e173bcf48292